api/routes/height_routes.py

```
@@ -0,0 +1,117 @@
+
+    if height is None:
+        return HTTPResponse(status=400, body=json.dumps({"error": "height missing"}))
+    try:
+        height_controller.save_height(height)
+        return json.dumps({"success": "saved height: " + height})
+    except Exception as e:
+        return HTTPResponse(status=500, body=json.dumps({"error": str(e)}))
+
+
+@height_server.route("/current")
+def current_height():
```

Ruff

Redefinition of unused `current_height` from line 9

(F811)